src/shmem_team.c

```
@@ -504,11 +504,11 @@ int shmem_internal_team_split_2d(shmem_internal_team_t *parent_team, int xrange,
     return 0;
 }
 
-int shmem_internal_team_destroy(shmem_internal_team_t *team)
+void shmem_internal_team_destroy(shmem_internal_team_t *team)
 {
 
     if (team == SHMEM_TEAM_INVALID) {
-        return -1;
+        return;
```

This is the only change I think would merit discussion. It used to return -1 but no error was really displayed. My question is if the team is invalid do we want to RAISE an error like we do below?

---

The copy of the spec I have laying around says 'If team compares equal to SHMEM_TEAM_INVALID, then no operation is performed'. I think this means you might be able to just return silently.